src/control/cmd/daos_agent/main.go

```diff
@@ -202,8 +202,10 @@ func parseOpts(args []string, opts *cliOptions, invoker control.Invoker, log *lo
 			// Create an additional set of loggers which append everything
 			// to the specified file.
 			log.WithErrorLogger(logging.NewErrorLogger("agent", f)).
+				WithNoticeLogger(logging.NewNoticeLogger("agent", f)).
 				WithInfoLogger(logging.NewInfoLogger("agent", f)).
-				WithDebugLogger(logging.NewDebugLogger(f))
+				WithDebugLogger(logging.NewDebugLogger(f)).
+				WithTraceLogger(logging.NewTraceLogger(f))
```

Comment:
The new log level  TRACE and NOTICE should be added in the comment section of the `control_log_mask` key of the file _utils/config/daos_agent.yml_